news/_posts/2020-12-23-ANH-release.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: AbstractAlgebra.jl 0.12.0, Nemo 0.19.0 and Hecke 0.9.0 released
```

Comment:
Maybe just "AbstractAlgebra" (without ".jl"), to be consistent with the other 2? (same below in the first sentence).